src/libs/Launcher.js

```diff
@@ -438,7 +438,8 @@ export default class Launcher {
           'size',
           'md5sum',
           'mime',
-          'trashed'
+          'trashed',
+          '_rev'
```

Comment:
I'm a bit surprised, I thought the `_rev` was *always* included in the response, whatever the `.select`?

Reply:
It seems so: https://github.com/cozy/cozy-stack/blob/master/web/files/files.go#L1487-L1488

Reply:
You're right, let's go for another fix!